### PR TITLE
Community updates: community panel display with markdown

### PR DESCRIPTION
**Pull request by Enderedne**
### Describe the background of your pull request

Display community updates in the community section. Added some minor enhancements

### Additional context

Related to PR #[1312](https://github.com/alkem-io/server/pull/1312)

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/components/Community/CommunitySection.tsx [hunk 1]
```diff
@@ -54,10 +56,16 @@ export const CommunitySection: FC<CommunitySectionProps> = ({
...
 
+  const updatesCountLabel = updates?.length ? `(${updates?.length})` : '';
   const tabList = [
     { name: 'members', label: 'Members', enabled: true },
-    { name: 'updates', label: 'Updates', enabled: isFeatureEnabled(FEATURE_COMMUNICATIONS) },
-    { name: 'discussion', label: 'Discussion', enabled: isFeatureEnabled(FEATURE_COMMUNICATIONS) },
+    { name: 'updates', label: `Updates ${updatesCountLabel}`, enabled: isFeatureEnabled(FEATURE_COMMUNICATIONS) },
+    {
+      name: 'discussion',
+      label: 'Discussion (Comming soon)',
```
Comment (hero101) [inline src/components/Community/CommunitySection.tsx:65]: `coming`

**Review by hero101 — COMMENTED** [review]
Waiting until #972 is merged. This branch seems to be dependant on that branch.

File: src/components/Community/CommunitySection.tsx (hunk 1 above)
Comment (Enderedne) [reply src/components/Community/CommunitySection.tsx]: Done.

File: src/components/Community/CommunitySection.tsx [hunk 2]
```diff
@@ -78,7 +86,35 @@ export const CommunitySection: FC<CommunitySectionProps> = ({
...
+                  {updates && (
+                    <AvatarsProvider users={users}>
+                      {detailedUsers => (
+                        <CommunityUpdatesView
+                          entities={{
+                            members: detailedUsers,
+                            messages: updates,
+                          }}
+                          actions={{
+                            onSubmit: (_: string) => {
```
Comment (hero101) [inline src/components/Community/CommunitySection.tsx:99]: What is the purpose of this handler?

**Review by hero101 — APPROVED** [review]

File: src/components/Community/CommunitySection.tsx (hunk 2 above)
Comment (Enderedne) [reply src/components/Community/CommunitySection.tsx]: Was intended to serve a placeholder if we enable admins to write posts from the community section, but we can drop it for now.

**Review by hero101 — APPROVED** [review]

**Pull request merged**